### PR TITLE
Fix TypeError: Cannot read property 'isGuest' of undefined

### PR DESCRIPTION
**Pull request by lyubomir**
We recently moved JSON Web Token (JWT) in redux so the related information is no longer available on `APP.tokenData`.

### CODE REVIEW

File: react/features/toolbox/actions.web.js [hunk 1]
```diff
@@ -218,28 +215,22 @@ export function showSharedVideoButton(): Function {
...
                 && UIUtil.isButtonEnabled(buttonName)
-                && show
                 && (!config.enableUserRolesBasedOnToken
-                        || !APP.tokenData.isGuest);
-
-        if (shouldShow) {
+                    || !getState()['features/jwt'].isGuest)) {
             dispatch(setToolbarButton(buttonName, {
-                hidden: !shouldShow
+                hidden: false
```
Comment (saghul) [inline react/features/toolbox/actions.web.js]: This function used to hide the button if `show` was false. Is that change intentional?

Comment (lyubomir) [inline react/features/toolbox/actions.web.js]: No, it didn't. The `dispatch` was executed only if `shouldShow` was `true`.

Comment (saghul) [inline react/features/toolbox/actions.web.js]: Oh, I see. Sorry for the noise!

Comment (saghul) [inline react/features/toolbox/actions.web.js]: At any rate, it's a bit weird we don't always dispatch, no?
